src/platform/packages/shared/kbn-esql-utils/src/utils/append_to_query.ts

```diff
@@ -88,7 +115,10 @@ export function appendWhereClauseToESQLQuery(
       if (matches) {
         const existingOperator = matches[1]?.trim().replace('`', '').toLowerCase();
         if (!['==', '!=', 'is not null', 'is null'].includes(existingOperator.trim())) {
-          return appendToESQLQuery(baseESQLQuery, `and ${fieldName}${operator}${filterValue}`);
+          return appendToESQLQuery(
+            baseESQLQuery,
+            `and ${fieldName} ${operator} ${filterValue}`.trim()
```

Comment:
The space here is necessary because the newly defined `getOperator` function doesn't return the post unary operator with a leading space like we had it before, this is also why couple of other test cases got impacted, this however does not impact the validity of the query generated